docs/installation/production/configuration/advanced/django_channels_layer.rst

```diff
@@ -0,0 +1,48 @@
+********************************
+Django Channels Layer (Optional)
+********************************
+
+**Last Updated:** Sep 2020
+
+Production installations that use the ``WebSockets`` and/or ``Bokeh Server`` functionality that comes with Tethys, require the a ``CHANNEL_LAYER``.
+
+Key Concepts
+============
+A ``CHANNEL_LAYER`` provides a backend where ``WebSocket`` messages can be stored and then accessed by  different app instances (ASGI processes).
+
+Adding a REDIS CHANNEL_LAYER
+============================
+Development installations make use of the default ``InMemoryChannelLayer``, however this layer is not meant for production (See `Django Channels documentation <https://channels.readthedocs.io/en/latest/topics/channel_layers.html#in-memory-channel-layer>`_). For production purposes, ``Django Channels`` suppports a ``REDIS CHANNEL LAYER``, however other ``CHANNEL_LAYERS`` can be configured. The following documentation demonstrates how to configure a ``REDIS CHANNEL LAYER``.
+
+First, install the ``channels_redis`` Python package in your Tethys conda environment.
+
+..
+
+    pip install channels_redis
```

Comment:
Let's have `channels_redis` installed in the Docker image by default so it is easy to enable.